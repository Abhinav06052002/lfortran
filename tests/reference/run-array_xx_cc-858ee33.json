{
    "basename": "run-array_xx_cc-858ee33",
    "cmd": "lfortran --continue-compilation --no-color {infile}",
    "infile": "tests/errors/array_xx_cc.f90",
    "infile_hash": "6fb257042311d57514493479dd6cd0ccea2d922d070e572dd398dfae",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "run-array_xx_cc-858ee33.stdout",
    "stdout_hash": "7c30da867d34c3f53085fa5826aac5eebd24829d11e7189553ee7d4a",
    "stderr": "run-array_xx_cc-858ee33.stderr",
    "stderr_hash": "118a37dda92a30d229d7a57a98eee08db8cbb8360560f0eddce4fbf0",
    "returncode": 1
}