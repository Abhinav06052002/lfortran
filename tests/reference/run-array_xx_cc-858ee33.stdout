(TranslationUnit
    (SymbolTable
        1
        {
            array_xx_cc:
                (Program
                    (SymbolTable
                        2
                        {
                            1_t_x:
                                (ExternalSymbol
                                    2
                                    1_t_x
                                    3 x
                                    t
                                    []
                                    x
                                    Public
                                ),
                            a:
                                (Variable
                                    2
                                    a
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Array
                                        (Integer 4)
                                        [((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 5 (Integer 4) Decimal))]
                                        FixedSizeArray
                                    )
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            a1:
                                (Variable
                                    2
                                    a1
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Array
                                        (Integer 4)
                                        [((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 3 (Integer 4) Decimal))]
                                        FixedSizeArray
                                    )
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            b:
                                (Variable
                                    2
                                    b
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Array
                                        (Integer 4)
                                        [((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 10 (Integer 4) Decimal))]
                                        FixedSizeArray
                                    )
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            c:
                                (Variable
                                    2
                                    c
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            i:
                                (Variable
                                    2
                                    i
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            iarx:
                                (Variable
                                    2
                                    iarx
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Array
                                        (Integer 4)
                                        [((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 3 (Integer 4) Decimal))]
                                        FixedSizeArray
                                    )
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            k:
                                (Variable
                                    2
                                    k
                                    []
                                    Local
                                    (IntegerConstant 3 (Integer 4) Decimal)
                                    (IntegerConstant 3 (Integer 4) Decimal)
                                    Save
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            size_a:
                                (Variable
                                    2
                                    size_a
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            str:
                                (Variable
                                    2
                                    str
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Array
                                        (String 1 4 () PointerString)
                                        [((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 3 (Integer 4) Decimal))]
                                        FixedSizeArray
                                    )
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            t:
                                (Struct
                                    (SymbolTable
                                        3
                                        {
                                            x:
                                                (Variable
                                                    3
                                                    x
                                                    []
                                                    Local
                                                    ()
                                                    ()
                                                    Default
                                                    (Integer 4)
                                                    ()
                                                    Source
                                                    Public
                                                    Required
                                                    .false.
                                                )
                                        })
                                    t
                                    []
                                    [x]
                                    Source
                                    Public
                                    .false.
                                    .false.
                                    []
                                    ()
                                    ()
                                ),
                            x1:
                                (Variable
                                    2
                                    x1
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Array
                                        (Integer 4)
                                        [((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 3 (Integer 4) Decimal))
                                        ((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 2 (Integer 4) Decimal))]
                                        FixedSizeArray
                                    )
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            x2:
                                (Variable
                                    2
                                    x2
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            x3:
                                (Variable
                                    2
                                    x3
                                    []
                                    Local
                                    (IntegerConstant 2 (Integer 4) Decimal)
                                    (IntegerConstant 2 (Integer 4) Decimal)
                                    Parameter
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            y:
                                (Variable
                                    2
                                    y
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (StructType
                                        2 t
                                    )
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    array_xx_cc
                    []
                    [(Assignment
                        (Var 2 a)
                        (Var 2 a)
                        ()
                    )
                    (Assignment
                        (Var 2 x2)
                        (IntegerUnaryMinus
                            (IntegerConstant 3 (Integer 4) Decimal)
                            (Integer 4)
                            (IntegerConstant -3 (Integer 4) Decimal)
                        )
                        ()
                    )
                    (Assignment
                        (Var 2 c)
                        (IntegerConstant 0 (Integer 4) Decimal)
                        ()
                    )
                    (GoToTarget
                        1
                        1
                    )
                    (Assignment
                        (Var 2 c)
                        (IntegerBinOp
                            (Var 2 c)
                            Add
                            (IntegerConstant 1 (Integer 4) Decimal)
                            (Integer 4)
                            ()
                        )
                        ()
                    )
                    (GoToTarget
                        2
                        2
                    )
                    (Assignment
                        (Var 2 c)
                        (IntegerBinOp
                            (Var 2 c)
                            Add
                            (IntegerConstant 2 (Integer 4) Decimal)
                            (Integer 4)
                            ()
                        )
                        ()
                    )
                    (GoToTarget
                        3
                        3
                    )
                    (Assignment
                        (Var 2 c)
                        (IntegerBinOp
                            (Var 2 c)
                            Add
                            (IntegerConstant 4 (Integer 4) Decimal)
                            (Integer 4)
                            ()
                        )
                        ()
                    )
                    (Print
                        (StringFormat
                            ()
                            [(Var 2 c)]
                            FormatFortran
                            (String -1 0 () PointerString)
                            ()
                        )
                    )
                    (If
                        (IntegerCompare
                            (Var 2 c)
                            NotEq
                            (IntegerConstant 7 (Integer 4) Decimal)
                            (Logical 4)
                            ()
                        )
                        [(ErrorStop
                            ()
                        )]
                        []
                    )
                    (Assignment
                        (Var 2 size_a)
                        (Var 2 size_a)
                        ()
                    )
                    (Assignment
                        (Var 2 size_a)
                        (Var 2 size_a)
                        ()
                    )
                    (Assignment
                        (Var 2 x3)
                        (IntegerConstant 1 (Integer 4) Decimal)
                        ()
                    )
                    (Print
                        (StringFormat
                            ()
                            [(Var 2 x3)]
                            FormatFortran
                            (String -1 0 () PointerString)
                            ()
                        )
                    )]
                )
        })
    []
)
